src/osd/OSD.cc

```diff
@@ -3200,16 +3200,18 @@ int OSD::shutdown()
   cct->get_admin_socket()->unregister_command("ops");
   cct->get_admin_socket()->unregister_command("dump_blocked_ops");
   cct->get_admin_socket()->unregister_command("dump_historic_ops");
-  cct->get_admin_socket()->unregister_command("dump_historic_ops_by_duration");
   cct->get_admin_socket()->unregister_command("dump_historic_slow_ops");
+  cct->get_admin_socket()->unregister_command("dump_historic_ops_by_duration");
```

Comment:
@xiexingguo i was just looking at the "dump_historic_ops_by_duration" line. because your change is increasing the inversion number.